### PR TITLE
Updates and fixes

### PR DESCRIPTION
**Pull request by ltagliaferri**
Signed-off-by: ltagliaferri 

Closing #167 #168 #188

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ You'll then navigate to `localhost:1313` within the web browser of your choice.
 
 If you identify something that is a major change, please file an [issue](https://github.com/chainguard-dev/edu/issues/new). If you identify a minor change like a typo that needs to be updated, or tech tooling that has a newer package, you are welcome to open a pull request for review from the team.
 
+### Date Format
+
+In each post's header, the date format should follow year-month-day as `YYYY-MM-DD`.
```
Comment (ltagliaferri) [inline README.md:33]: We should eventually add a whole section on the header to the README

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/how-to-disable-policy-enforcement.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: "How to disable policy enforcement"
+type: "article"
+description: "Quickly handle incident responses in Chainguard Enforce"
+date: 2022-11-29T08:49:31+00:00
+lastmod: 2022-11-29T08:49:31+00:00
+draft: true
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/how-to-disable-policy-enforcement.md:7]: This is a draft and is not complete yet 😄

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md [hunk 3]
```diff
@@ -15,7 +15,9 @@ toc: true
...
 
-Chainguard Enforce for Kubernetes allows users to create their own security policies that they can enforce in their clusters. Here are a few example policies to help you get started. You may also want to review the [Sigstore Policy Controller documentation](https://docs.sigstore.dev/policy-controller/overview).
+Chainguard Enforce for Kubernetes allows users to create their own security policies that they can enforce in their clusters. Here are a few example policies to help you get started. You can also review the [policy cataglogue](https://console.enforce.dev/policies/catalog) in the Chainguard Enforce Console. 
+
+You may also review the [Sigstore Policy Controller documentation](https://docs.sigstore.dev/policy-controller/overview), and in particular you may review how to admit images through the cluster image policy, review the Policy Controller [Admission of images documentation](https://docs.sigstore.dev/policy-controller/overview/#admission-of-images).
```
Comment (SharpRake) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-policy-examples.md:20]:
```suggestion
You may also review the [Sigstore Policy Controller documentation](https://docs.sigstore.dev/policy-controller/overview). In particular, we encourage you to review the Policy Controller documentation relating to the [Admission of images](https://docs.sigstore.dev/policy-controller/overview/#admission-of-images) to learn how to admit images through the cluster image policy.
```

This sentence just felt kind of long

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-ui/index.md [hunk 4]
```diff
@@ -17,7 +17,9 @@ Security policies in Chainguard Enforce ensure that our development teams are de
...
+In the Chainguard Enforce Console, we have a [policy cataglogue](https://console.enforce.dev/policies/catalog) that provides example policies that you can use right away or modify for your use case. You may also wish to check out our page on [Chainguard Enforce Policy Examples](../chainguard-enforce-policy-examples). If you would like to use the CLI to work with policies, review our [`chainctl` policy docs](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainctl-docs/chainctl_policies_apply/) and review our guidance on [how to manage policies with `chainctl`](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-cli/).
+
+For additional policy guidance, you may also review the [Sigstore Policy Controller documentation](https://docs.sigstore.dev/policy-controller/overview), and in particular you may review how to admit images through the cluster image policy, review the Policy Controller [Admission of images documentation](https://docs.sigstore.dev/policy-controller/overview/#admission-of-images).
```
Comment (SharpRake) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-ui/index.md:22]:
```suggestion
For additional policy guidance, you may also review the [Sigstore Policy Controller documentation](https://docs.sigstore.dev/policy-controller/overview). To learn how to admit images through the cluster image policy, review the Policy Controller documentation relating to the [Admission of images](https://docs.sigstore.dev/policy-controller/overview/#admission-of-images).
```

Again just smoothing this out a tad.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-ui/index.md [hunk 5]
```diff
@@ -43,24 +45,15 @@ If your table indicates that there are current violations, you can click on the
...
 
-Whether you choose to begin with the **Custom** or **Signature** policy, a new sidebar will be displayed that has fields to choose the **Group**, and write a **Description** for your new policy. You can also edit the YAML file itself. 
-
-In the **Signature** policy, you'll have an additional field for the policy's **Name**, a field for **Image**, and an option to add more images. 
-
-![Create a Chainguard Enforce Signature policy](signature-policy.png)
+Whether you choose to begin with a **Custom** policy or another option from our catalogue, a new sidebar will be displayed that has fields to choose the **Group**, and write a **Description** for your new policy, and a YAML file, which you can edit.
 
-Fill in the fields for this policy until you are satisfied. In the policy below, we are requiring signatures for all Docker and GitHub container images. We have associated it with the `chainguard-demos` group. 
+Fill in the fields for this policy based on your policy enforcement goals. In the policy below, we are requiring signatures for all Docker and GitHub container images. We have associated it with the `chainguard-demos` group.
```
Comment (SharpRake) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-ui/index.md:56]: I really like this change!

**Review by SharpRake — COMMENTED** [review]
These changes are great! I added a few minor suggestions, but lmk if you think they're unnecessary.

**Review by SharpRake — APPROVED** [review]
Looks great to me!

**Pull request merged**